docs/docs/reference/contextual/extension-methods.md

```diff
@@ -111,11 +113,23 @@ extension (ss: Seq[String]):
   def longestString: String = longestStrings.head
 ```
 
+The same looks as follows with braces (note that you can still use an intended region for the methods):
```

Comment:
Typo:

```suggestion
The same looks as follows with braces (note that you can still use an indented region for the methods):
```

Though I'd make the whole sentence a bit clearer:

```suggestion
The same can be written with braces as follows (note that indentation-based syntax can still be used inside braces):
```

Reply:
I like the first part, but think it makes more sense to reuse the same terminology as already mentioned.